lib/webserver/WebServer.js

```diff
@@ -616,6 +616,24 @@ const WebServer = function (options) {
         res.json(self.lastMap.parsedData)
     });
 
+    this.app.get("/api/map/raw/latest", function (req, res) {
+        res.status(200).send(self.lastMap.rawData)
+    });
+
+    this.app.get("/api/flolevac/setup", function (req, res) {
+        self.vacuum.getCurrentStatus(function (err, data) {
+            if (err) {
+                res.status(500).send(err.toString());
+            } else {
+                res.json({
+                    token: self.vacuum.token.toString("hex"),
```

Comment:
is it good idea to make "token" available to anybody with local network access?

Reply:
You should ask that the person who wrote the function right under this one ;)